s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3WireMockBase.scala

```diff
@@ -35,19 +36,35 @@ abstract class S3WireMockBase(_system: ActorSystem, _wireMockServer: WireMockSer
 
   val body = "<response>Some content</response>"
   val bucketKey = "testKey"
+  val bucketKeyForRanged = "testKeyForRanged"
```

Comment:
I see what you mean. Then could it be a good idea to reset the wiremock server mappings after each test instead? This way we'll avoid unwanted mappings to mess with future tests